components/authz-service/server/v2/policy_refresher.go

```diff
@@ -299,3 +299,17 @@ func (refresher *policyRefresher) getRuleMap(ctx context.Context) (map[string][]
 func pretty(vsn api.Version) string {
 	return fmt.Sprintf("IAM v%d.%d", int32(vsn.Major), int32(vsn.Minor))
 }
+
+type mockPolicyRefresher struct{}
```

Comment:
❓ Can we defined this in `server/v2/policy_test.go` instead? Won't make it into prod code that way. (Not that it does much harm here, but... just for the sake of it)